afs/afs-ext-base/src/main/java/com/powsybl/afs/ext/base/AbstractScript.java

```diff
@@ -7,31 +7,33 @@
 package com.powsybl.afs.ext.base;
 
 import com.google.common.io.CharStreams;
+import com.powsybl.afs.OrderedDependencyManager;
 import com.powsybl.afs.ProjectFile;
 import com.powsybl.afs.ProjectFileCreationContext;
 import com.powsybl.afs.storage.events.AppStorageListener;
 import com.powsybl.afs.storage.events.NodeDataUpdated;
 import com.powsybl.afs.storage.events.NodeEvent;
 import com.powsybl.afs.storage.events.NodeEventType;
+import org.apache.commons.lang3.StringUtils;
 
 import java.io.*;
 import java.nio.charset.StandardCharsets;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Objects;
+import java.util.*;
+import java.util.stream.Collectors;
 
 /**
  * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
  */
-public abstract class AbstractModificationScript extends ProjectFile implements StorableScript {
```

Comment:
To avoid compilation failure, we should keep the old class and mark it as deprecated